docs/catalog/cloud/levarage-automation-non-prod-environment-recreate.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: cloud
+tags: 
+ - devops
```

Comment:
please include `cloud` in the list of tags

Reply:
Added tag